### DISCUSSION TITLE
Problemas com a publicação

### DISCUSSION THREAD

**Discussion by luizgoebel** [balta-io/forum#223] [Tira Dúvidas] [answered]
o erro esta em anexo assim como os fontes, fiz várias tentativas e comparação de código e não obtive sucesso

![CsProj](https://user-images.githubusercontent.com/72111314/212478038-3be5e9ef-8f29-4817-b3e0-e9d0ac1187d6.png)
![ErroPipeLine](https://user-images.githubusercontent.com/72111314/212478039-68e5d6ff-b033-47f1-8a67-537cd9fafdba.png)
![NugetAPI](https://user-images.githubusercontent.com/72111314/212478040-3dd474f5-911e-4a84-a032-cff04c831a75.png)
![MainYaml](https://user-images.githubusercontent.com/72111314/212478041-130d52a4-a23e-4f20-9b0e-dea8cae1fdf4.png)

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado:
https://github.com/balta-io/forum/discussions/84

**Reply by luizgoebel** [reply]
Não ficou claro a solução, fiz tentativas como as da postagem, porém sem sucesso

**Reply by andrebaltieri** [reply]
@luizgoebel boa tarde, quais tentativas você fez e quais erros obteve? Assim consigo te ajudar melhor.

**Reply by leandrolevv** [reply]
@luizgoebel, por algum motivo a chave NUGET_TOKEN não funcionou no meu, e tive que improvisar (pode ser o seu caso):
Criei um Personal Access Token, com permissão para escrever packages:
![image](https://user-images.githubusercontent.com/62189825/213063148-8fa4a54f-11ae-411e-a7cf-e85e739b73e3.png)
E criei uma nova secret para ela:
![image](https://user-images.githubusercontent.com/62189825/213063202-766efd81-9826-4e3d-8de4-56471b1af272.png)

**Comment by luizgoebel** [comment]
Obrigado a ajuda de todos e pela paciência, é muito bom saber que é uma comunidade unida.
Deu certo conforme imagem abaixo:

![image](https://user-images.githubusercontent.com/72111314/213173071-c43f05ad-faf6-4077-9317-8fadabd0c1d4.png)

**Reply by andrebaltieri** [reply]
Parabéns!!! 🚀
